app/src/main/java/uc/edu/klopfsea/groupscheduler/model/Event.kt

```diff
@@ -17,31 +17,28 @@ data class Event(
 
     // Sets the time
     fun setTime(t: Int) {
-        timeSpecification = if (t == 0)
-            "After $time $period"
-        else if (t == 1)
-            "Before $time $period"
-        else
-            "All Day"
+        timeSpecification = when (t) {
+            0 -> "After $time $period"
+            1 -> "Before $time $period"
+            else -> "All Day"
+        }
     }
 
     // formats output
     fun getFormatedDate(): String {
         var s = date.split('-')
-        var str = ""
+        var formattedOutput = ""
```

Comment:
renamed variable str to formattedOutput for easier understandability

Reply:
I'd go one step further.  What is being outputted here?  Use that to name the variable.